client/components/settings-group-card/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import FormSectionHeading from 'components/forms/form-section-heading';
+
+const SettingsGroupCard = ( { heading, children } ) => {
```

Comment:
This one too, this is not about the 5-lines JS component, its about all the style associated with it (look at its `style.scss` file). Would it help if we change its name? `SettingsGroupCard` is the best I could come up with.